client/blocks/eligibility-warnings/index.tsx

```diff
@@ -89,7 +87,7 @@ export const EligibilityWarnings = ( {
 			'eligibility-warnings__placeholder': isPlaceholder,
 			'eligibility-warnings--with-indent': showWarnings,
 			'eligibility-warnings--blocking-hold': hasBlockingHold( listHolds ),
-			'eligibility-warnings--without-title': ! title && ! primaryText,
+			'eligibility-warnings--without-title': context !== 'plugin-details',
```

Comment:
Might want to clean this up, just noticed we were now leveraging the title / primary text, added this as a temp fix.